### PR TITLE
Grade Buttons from harmony

### LINKED ISSUE #788: Grade Buttons from harmony

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Refactor
-   [x] Feature

## Related issues

Resolve #788

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
https://github.com/user-attachments/assets/7e176cde-e959-498f-b736-f375d702f310

File: src/components/SectionFeedback/SectionFeedback.tsx [hunk 1]
```diff
@@ -164,10 +163,7 @@ export const SectionFeedback = ({ section, isEditable, hasTasks }: SectionFeedba
                             <SectionFeedbackHireBadge hire={section.hire} />
                             {section.grade && (
                                 <>
-                                    {tr('Grade:')}{' '}
-                                    <GradeButton type="button" matching>
-                                        {section.grade}
-                                    </GradeButton>
+                                    {tr('Grade:')} <Button type="button" text={section.grade} />
```
Comment (tsumo) [inline src/components/SectionFeedback/SectionFeedback.tsx:166]: Unclickable buttons. Looks bad from UX perspective

**Comment by Katrin-kudryash** [comment]
w

**Review by tsumo — APPROVED** [review]

**Comment by Katrin-kudryash** [comment]
q

**Comment by Katrin-kudryash** [comment]
Untitled

File: src/components/Section/Section.module.css [hunk 2]
```diff
@@ -2,10 +2,23 @@
     display: flex;
     flex-direction: column;
     gap: var(--gap-m);
+    /* width: 900px; */
```
Comment (asabotovich) [inline src/components/Section/Section.module.css:5]: remove?

File: src/components/SectionResults/SectionResults.tsx [hunk 3]
```diff
@@ -1,60 +1,66 @@
...
 interface SectionResultsProps {
     passedSections: SectionWithSectionType[];
+    className?: string;
 }
 
-const sectionColumnWidth = '250px';
-const interviewerColumnWidth = '150px';
-
-export const SectionResults = ({ passedSections }: SectionResultsProps): JSX.Element | null => {
+export const SectionResults = ({ passedSections, className }: SectionResultsProps): JSX.Element | null => {
```
Comment (asabotovich) [inline src/components/SectionResults/SectionResults.tsx:16]: ``FC<SectionResultsProps>`` instead of return type?

**Review by asabotovich — APPROVED** [review]

**Pull request merged**